UPGRADE-4.3.md

```diff
@@ -55,6 +55,35 @@ EventDispatcher
 ---------------
 
  * The signature of the `EventDispatcherInterface::dispatch()` method should be updated to `dispatch($event, string $eventName = null)`, not doing so is deprecated
+
+   You have to swap arguments when calling `dispatch()`:
+
+   Before:
+   ```php
+   $this->eventDispatcher->dispatch(Events::My_EVENT, $event);
+   ```
+
+   After:
+   ```php
+   $this->eventDispatcher->dispatch($event, Events::My_EVENT);
+   ```
+
+   If your bundle of package need to provide FC/BC compat with the previous way of using the dispatcher, you can use `Symfony\Component\EventDispatcher\LegacyEventDispatcherProxy::decorate()` to ease upgrade:
```

Comment:
`If your bundle or package needs to provide compatibility with the previous way of using the dispatcher [...] upgrades:`